### DISCUSSION TITLE
Using same view for devices

### DISCUSSION THREAD

**Discussion by killerbeat12** [frangoteam/FUXA#1157] [Q&A]
I know it is possible to have a same view for tags (like Valve A and B) but i have like 10 devices with same tags, is it possible to change the device name and that is shows the tag values from the selected device?

**Comment by unocelli** [comment]
Hi, you can use 'placeholder' or 'internal' device

**Reply by henjoe** [reply]
hi @unocelli do you have sample instruction on how to use the place holder? I found this (https://github.com/frangoteam/FUXA/wiki/HowTo-use-same-view)  tutorial but creating an internal tags for every devices is a huge work. I just don't know maybe a "placeholder" will work better on this kind of scenario.

**Comment by killerbeat12** [comment]
Yes i agree, i am using now the internal tags which are working pretty fine but it is also a lot of engineering work to assign 20 internal tags to multiple devices.. if you could give a small explanation how to use the placeholder should be nice.. thanks in advance appreciate it.

**Reply by henjoe** [reply]
There is already a placeholder my friend, you dont have to create 20 tags, just use place holder.

**Reply by killerbeat12** [reply]
Hi Henjoe, thanks for replying, could you show me an example please?

**Reply by henjoe** [reply]
Fyr.

https://youtu.be/nBlYThk2304?si=oJIoi6jUYqYFy8En

**Reply by killerbeat12** [reply]
mhh alright, this is same functionality as internal tags, i am more looking to replace a part of the tag with the placeholder. all my tags have a devicename and signal name. i would like to replace the devicename with a placeholder but with the current placeholder you could only replace a whole tag.

Example:
tag1 = Device1_Speed
tag2 = Device2_Speed

Placeholder = @Device
@Device = Device1

Assign placeholder to a object like @Device + "_Speed" (result will be Device1_Speed for the object)

**Reply by henjoe** [reply]
> mhh alright, this is same functionality as internal tags, i am more looking to replace a part of the tag with the placeholder. all my tags have a devicename and signal name. i would like to replace the devicename with a placeholder but with the current placeholder you could only replace a whole tag.
> 
> Example:
> tag1 = Device1_Speed
> tag2 = Device2_Speed
> 
> Placeholder = @Device
> @Device = Device1
> 
> Assign placeholder to a object like @Device + "_Speed" (result will be Device1_Speed for the object)


I see, I think that is not yet possible as of the moment. And may take time before it get there.
